docs/developer/app-store/apps/emails-and-messages/overview.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Overview
+sidebar_position: 1
+---
+
+# Introduction
+
+Emails and messages (EAM) application is responsible for sending emails and messages to the customers. It is connected to the Saleor API via webhooks, which are used to notify the application about the events. Delivery of the messages is done by the external providers, which are configured in the application.
+
+## Suported providers
+
+Currently, we are supporting:
+
+- [Sendgrid](sendgrid) - Create templates in the Sengrid visual editor and send them to your customers
+- [SMTP](smtp) - Use your own SMTP server to send emails. Email templates are created using MJML and handlebars.
+
+## Supported events
+
+The application is listening to the following events:
+
+- `account_confirmation`
+- `account_password_reset`
+- `account_change_email_request`
+- `account_change_email_confirm`
+- `account_delete`
+- `account_set_customer_password`
+- `invoice_ready`
+- `order_confirmed`
+- `order_fulfillment_confirmation`
+- `order_fully_paid`
+- `order_canceled`
+- `send_gift_card`
+
+## Configuration
+
+The application can be simultaneously configured to use multiple providers. You'll be able to:
+
+- create test configurations before using them in production
+- use different providers for different events
+- use different providers for different channels
+
+### Multichannel
+
+Each provider can be configured to work with specific channels. The configurations by default are used for all channels. It can be changed in the configuration details to:
+
+- use only the specific channels
+- use all channels except the specific ones
+
+## Development
```

Comment:
probably worth linking https://docs.saleor.io/docs/3.x/developer/app-store/development

and leave here only app specific stuff